pom.xml

```diff
@@ -344,12 +344,24 @@
       <version>3.2.1</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter</artifactId>
+      <version>5.7.2</version>
+      <scope>test</scope>
+    </dependency>
     <dependency>
```

Comment:
JUnit 4 should be removed.

Reply:
Oh, I thought we'd have to keep the JUnit 4 dependency until all tests were migrated to JUnit 5.
I guess `junit-vintage-engine` doesn't require that.

At least that's the impression the docs gave me.

> The JUnit Platform can run JUnit 4 based tests as long as you configure a testImplementation dependency on JUnit 4 and a testRuntimeOnly dependency on the JUnit Vintage TestEngine implementation similar to the following.
> 
> ```gradle
> dependencies {
>     testImplementation("junit:junit:4.13")
>     testRuntimeOnly("org.junit.vintage:junit-vintage-engine:5.7.2")
> }
> ```
> \- https://junit.org/junit5/docs/current/user-guide/#running-tests-build-gradle-engines-configure

Just tried now and it worked perfectly fine without the JUnit 4 dependency, though. 🤔